Dockerfile

```
@@ -1,5 +1,15 @@
-FROM node:20-alpine
-RUN apk add g++ make python3 dumb-init
+FROM node:20-bullseye
```

```suggestion
FROM node:20-bookworm
```
`bookworm` is latest stable version of Debian

---

Might even get away with slim version
```suggestion
FROM node:20-bookworm-slim
```

---

Yep. Slim worked. I also used the RUN command provided by Temporal for slim images.